datastream-server/src/main/java/com/linkedin/datastream/server/zk/ZkAdapter.java

```diff
@@ -1075,7 +1075,8 @@ public int cleanUpOrphanConnectorTasks(boolean cleanUpOrphanTasksInConnector) {
       connectorTaskList.remove(KeyBuilder.DATASTREAM_TASK_LOCK_ROOT_NAME);
 
       if (connectorTaskList.size() > 0) {
-        LOG.warn("Found orphan tasks: {} in connector: {}", connectorTaskList, connector);
+        LOG.warn("Found {} number of orphan tasks: {} in connector: {}", connectorTaskList.size(),
```

Comment:
nit pick: `Found 10 number of orphan task` does not sound right. How about `Found 10 orphan tasks:`?